main.cpp

```diff
@@ -199,12 +371,39 @@ struct CarWash //                                   1) define an empty struct fo
     3) define the release of a sound
  */
 
+struct Adsr       
+{
+    //has attack
+    int attack = 127; //3) member variables with relevant data types.
+    //has decay
+    int decay = 127; //3) member variables with relevant data types.
+    //has sustain
+    int sustain = 23; //3) member variables with relevant data types.
+    //has release
+    int release = 27; //3) member variables with relevant data types.
+    //has amount
+    int amount = 127; //3) member variables with relevant data types.
```

Comment:
What kind of information are these properties supposed to represent?
- [ ] revise.  pick properties that provide information and name them accordingly.

Reply:
this is my adsr lass to provide the following properties to the adsr section. int attack is the value of the attack parameter for example.